native.go

```diff
@@ -0,0 +1,130 @@
+package libovsdb
+
+import (
+	"fmt"
+)
+
+// NativeAPI is an API that offers functions to interact with libovsdb without
+// having to handle it's internal objects. It uses a DatabaseSchema to infer the
+// type of each value and make translations.
+// OvsMaps are translated to go maps with specificif key and values. I.e instead of
+//	having to deal with map[interface{}][interface{}], the user will be able to
+//	user  map[string] string (or whatever native type can hold the column value)
+// OvsSets, same as OvsMap.
+//
+// OvsUUID are translated to and from strings
+// If the column type is an enum, the native type associated with the underlying enum
+// type is used (e.g: string or int)
+// Also, type checkings are done. E.g: if you try to put an integer in a column that has
+// type string, the API will refuse to create the Ovs object for you
+type NativeAPI struct {
+	schema DatabaseSchema
+}
+
+// NewNativeAPI returns a NativeAPI
+func NewNativeAPI(schema DatabaseSchema) NativeAPI {
```

Comment:
why returned type is not pointer? (does this have performance/memory improvement, or simply you preferred way...?

Reply:
You're right, it should return a pointer. Will update. Thanks.